grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ng.html

```diff
@@ -0,0 +1,34 @@
+<div *ngIf="flowRunning$ | async" class="in-progress">
+  {{ (args$ | async).query }}
+</div>
+
+<div *ngIf="flowCompleted$ | async" class='results'>
+  <div *ngIf="resultTable$ | async as resultTable">
+    <div class="results-query">
+      <div class="results-query-label">Query:</div>&nbsp;
```

Comment:
Are you sure this nbsp works as intended (since there is whitespace surrounding it). Also, instead of using `<div>`s with `display: inline` in the style, prefer using `<span>` which is `display: inline` per default.

Reply:
Basically I want some margin around the label and the query. So maybe I should use the margin style?

Didn't know about `span`s, I'm gonna use them from now on.

Reply:
Yes, please use explicit styling (padding/margin) instead of repurposing special characters. Haven't seen &nbsp; for styling in a looooong time :D

Reply:
Hmmmmm: https://github.com/google/grr/blob/22db4ed1a6020437fff7b84b5f32ece23db9b9c9/grr/server/grr_response_server/gui/ui/components/flow_details/helpers/file_results_table.ng.html#L17

Reply:
That's the intended use: There should be a space between the path and the copy button, but the copy button should stay on the same line without line-break: non-breaking space.

Reply:
Indeed that's the only one. Should we use styling there too?

Reply:
Oh, or maybe you mean we actually need the non-breaking feature, and we won't be getting that with just setting the margin style?